### PR TITLE
Cypress - Catching specific uncaught exception: application error front page Arabic AMP

### PR DESCRIPTION
**Pull request by LilyL0u**
Temporary fix for https://github.com/bbc/simorgh/issues/9138

**Overall change:**

Screenshot 2021-05-26 at 16 30 07

This error regularly fails our scheduled live E2Es and causes noise on the simorgh alerts channel.  We think it could be a problem with google's ad solution, and this has been escalated externally. In the meantime, we will catch this error. We will remove the catch when google resolve the issue.

I have placed the catch only in this test, so that if we get application errors anywhere else in the code, we are still alerted to them.

**Code changes:**

Used code from Cypress documentation https://docs.cypress.io/api/events/catalog-of-events#To-conditionally-turn-off-uncaught-exception-handling-unhandled-promise-rejections

I also added the done() outside the error catching clause as otherwise I get an error saying done() callback is not resolved. Let me know if this is not the way I should be doing this.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

Tests pass on live where the error occurs when there are no ads. We will need to see how it goes in AWS as I cannot check the tests in the EEA where there are ads and the error occurs.

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]
👍 looks like the yarn.lock changes are desirable - they seem to be a side effect of this commit which updated package.json but not the lock file: https://github.com/bbc/simorgh/pull/9153/commits/50aff3c99c4d40d1bea9e133340918c78f9bfbeb

File: cypress/integration/pages/frontPage/tests.js [hunk 1]
```diff
@@ -18,7 +18,16 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
           serviceHasPublishedPromo(service) &&
           Cypress.env('APP_ENV') === 'live'
         ) {
-          it('individual promo should link to corresponding article pages and back navigation should link to frontpage', () => {
+          it('individual promo should link to corresponding article pages and back navigation should link to frontpage', done => {
+            // This is to catch an application error that keeps failing live E2Es
+            // We are waiting for a response from Google to find a fix
+            // And in the meantime are stopping this error failing the tests
+            // eslint-disable-next-line no-unused-vars
```
Comment (andrewscfc) [inline cypress/integration/pages/frontPage/tests.js:26]: @LilyL0u could you link the issue you were going to create in the comment, just will make it easier to resolve this if it gets forgotten and someone comes across this later.

Also, nice work getting this to work in a Single test, I gave up and did it globally in the example I send you this is much more focused 👍

Comment (LilyL0u) [reply cypress/integration/pages/frontPage/tests.js:26]: Linked. Thanks :)

Comment (andrewscfc) [reply cypress/integration/pages/frontPage/tests.js:26]: Sorry, I meant in the code comment, did you forget to push?

Comment (LilyL0u) [reply cypress/integration/pages/frontPage/tests.js:26]: OH i see what you mean

Comment (LilyL0u) [reply cypress/integration/pages/frontPage/tests.js:26]: I thought you meant a github comment. I have now added the comment in the code.

Comment (andrewscfc) [reply cypress/integration/pages/frontPage/tests.js:26]: Thanks @LilyL0u

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**